src/transactions/multisignatureSend.js

```diff
@@ -34,7 +34,7 @@ import { prepareTransaction } from './utils';
  * @return {string}
  */
 
-export default function createTransaction(
+export default function multisignatureSend(
```

Comment:
This breaks the pattern of beginning with an imperative verb. `sendFromMultisignatureAccount`?

Reply:
Alright, even though it should be temporarily and multisignature transactions should be supported for every transaction. Therefore this should be deprecated quite quickly